src/main/java/com/epam/spring/boot/cargodeliverysystem/repository/imp/CityRepositoryImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.spring.boot.cargodeliverysystem.repository.imp;
+
+import com.epam.spring.boot.cargodeliverysystem.model.City;
+import com.epam.spring.boot.cargodeliverysystem.repository.CityRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
```

Comment:
Use @Repository annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers

Reply:
fixed. Changed to @Repository